label_studio/ml_models/models.py

```diff
@@ -67,6 +67,10 @@ class Meta:
 
     prompt = models.TextField(_('prompt'), null=False, blank=False, help_text='Prompt to execute')
 
+    model_provider_connection = models.ForeignKey(
+        ModelProviderConnection, related_name='model_versions', on_delete=models.SET_NULL, null=True
+    )
+
```

Comment:
is there a way to link the `provider` attr in ThirdPartyModelVersion to this object? migration to make it a @property?

Reply:
Link provider to ModelVersion? Not sure what you mean or what you want to achieve here